### PR TITLE
Upgrade `ecs-updater` dependencies

### PR DESCRIPTION
**Pull request by jpmcb**
**Issue number:**

Related to https://github.com/bottlerocket-os/bottlerocket-sdk/pull/87
and captures using Go 1.19.4

**Description of changes:**

```
- Upgrades aws sdk
- Upgrades bottlerocket sdk image

Signed-off-by: John McBride <jpmmcb@amazon.com>
```

**Testing done:**

Once SDK pr merges and releases, will do integration tests.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by webern — APPROVED** [review]

**Comment by stmcginnis** [comment]
Looks like the SDK needs to be updated now.

I'm also seeing `x/net` and friends. Can we get the workaround for the containerd > go-restful > x/* updates here too?

**Comment by jpmcb** [comment]
Yes - @yeazelm was going to take a stab at an upgrade PR to replace this one!!

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # SHELL is set as bash to use some bashisms.
 SHELL = bash
 
-BOTTLEROCKET_SDK_VERSION = v0.28.0
+BOTTLEROCKET_SDK_VERSION = v0.29.0
```
Comment (webern) [inline Makefile:4]:
```suggestion
BOTTLEROCKET_SDK_VERSION = v0.30.2
```

**Review by webern — APPROVED** [review]

**Comment by jpmcb** [comment]
Closed by https://github.com/bottlerocket-os/bottlerocket-ecs-updater/pull/114

**Pull request closed without merging**